.circleci/config.yml

```diff
@@ -33,19 +33,24 @@ commands:
 jobs:
   lint:
     executor: node
+    resource_class: large
     steps:
       - checkout-and-dependencies
       - run: yarn lint
 
   tests:
     executor: node
+    resource_class: large
     steps:
       - checkout-and-dependencies
-      - run: yarn test --coverage --runInBand
+      # We use workerIdleMemoryLimit to work around a memory issue with node.
+      # See https://github.com/facebook/jest/issues/11956
+      - run: yarn test --coverage --logHeapUsage -w=4 --workerIdleMemoryLimit=1.5G
```

Comment:
Using `--workerIdleMemoryLimit` doesn't look necessary right now, but this is a security in case our tests grow bigger. I checked that this works (we clearly see in the output that some workers are restarted).

I keep `logHeapUsage` for debugging reasons too.